test/TesterInternal/MembershipTests/DynamoDBMembershipTableTest.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Orleans;
+using Orleans.Messaging;
+using Orleans.Runtime;
+using Orleans.Runtime.MembershipService;
+using System.Threading.Tasks;
+using UnitTests.StorageTests.AWSUtils;
+using Xunit;
+
+namespace UnitTests.MembershipTests
+{
+    /// <summary>
+    /// Tests for operation of Orleans Membership Table using AWS DynamoDB - Requires access to external DynamoDB storage
+    /// </summary>
+    public class DynamoDBMembershipTableTest : MembershipTableTestsBase, IClassFixture<DynamoDBStorageTestsFixture>
```

Comment:
I believe one way of doing this would be to try to reach *DynamoDb* with the default connection and if it fails, skip the tests using `SkippableFact`. The explanation could be the database was unreachable, likely not present.

However, this is not my call. It is only my concern that I would like to see yellow for tests with unmet environment setup instead of red and without *being in the know about it*. Maybe this could be put also so that I have a creeping nagging feeling we're accumulating test code that's mostly red to most of the people and has duplication in code and in structure.

Then optinally it would be nice to have a way to override environment setup per test-site (i.e. if I have a differing setup on my rig). Not impossible, but not something we have at the moment either.